docs/how-tos/train-deploy-ml.md

```diff
@@ -306,27 +331,42 @@ To expand the image, click on the expand side menu arrow in the corner of the im
 {{<gif webm_src="/services/data/label-dog-big.webm" mp4_src="/services/data/label-dog-big.mp4" alt="Add a bounding box around the dog in an image in a big menu">}}
 
 Repeat this with all images.
-To see all the images that have bounding boxes, you can filter your dataset by selecting the label from the **Bounding box labels** dropdown in the **Filters** menu.
 
+You can add one or more bounding boxes for objects in each image.
+{{< /expand >}}
+
+Determining a descriptive state about an image.
+For example, you may wish to know whether an image of a food display is `full`, `empty`, or `average` or whether the quality of manufacturing output is `good` or `bad`.
+In this case, add tags to describe your images.
+
+{{< expand "For instructions to add tags, click here." >}}
+To tag an image, click on an image and select the **Image tags** mode in the menu that opens.
+Add one or more tags to your image.
+
+{{<gif webm_src="/services/data/tag-star.webm" mp4_src="/services/data/tag-star.mp4" alt="Tag image with a star label">}}
+
+If you want to expand the image, click on the expand side menu arrow in the corner of the image.
+
+Repeat this with all images.
 {{< /expand >}}
 
 {{% /tablestep %}}
 {{% tablestep link="/services/data/dataset/" %}}
-**3. Create a dataset**
+**7. Organize data into a dataset**
 
-A [dataset](/services/data/dataset/) allows you to conveniently view, work with, and train an ML model on a collection of images.
+To train a model, your images must be in a dataset.
 
-Use the interface on the **DATA** tab (or the [`viam dataset data add` command](/cli/#dataset)) to add all images you want to train the model on to a dataset.
+Use the interface on the **DATA** tab to add your labeled images to a dataset.
```

Comment:
```suggestion
Use the interface on the **DATA** tab to add your labeled or tagged images to a dataset.
```
[opt] maybe we are using the term “labeled” loosely since the app itself conflates them

Reply:
Yep I think a tag is a label. Tagging is therefore still labelling